.devcontainer/adx/create-adx.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+# Display Help
+Help()
+{
+   echo "Create data explorer cluster for development purpose."
+   echo
+   echo "Syntax: create-adx.sh [-u|h]"
+   echo "options:"
+   echo "u     Unique name used to create resources."
+   echo "h     Print this Help."
+   echo
+}
+
+while getopts u:h flag;
+do
+    case "${flag}" in
+        h) Help
+           exit;;
+        u) unique=${OPTARG};;
+    esac
+done
+
+if [ ! "$unique" ]; then
+    echo "Please add unique name argument."
+    exit;
+fi
+
+# Resource Names
+RESOURCE_GROUP_LOCATION="westeurope"
+RESOURCE_GROUP_NAME="rg-k2-$unique-dev"
+ADX_CLUSTER_NAME="adxk2$unique"
+ADX_DB_NAME="dev"
+STORAGE_ACCOUNT_NAME="stk2$unique"
+STORAGE_CONTAINER_NAME="scripts"
+SERVICE_PRINCIPAL_NAME="sp-k2-$unique"
+
+# Create Resource Group
+RESOURCE_GROUP_SCOPE=$(az group create --location $RESOURCE_GROUP_LOCATION --resource-group $RESOURCE_GROUP_NAME --query id -o tsv)
+
+# Create Data Explorer Cluster
+az extension add -n kusto
+echo "$(tput setaf 2)[1/8] Creating Data Explorer Cluster. Please be patient. Long running operation (10 minutes).$(tput setaf 7)"
+az kusto cluster create \
+    --cluster-name $ADX_CLUSTER_NAME \
+    --sku name="Dev(No SLA)_Standard_D11_v2" tier="Basic" capacity=1 \
+    --resource-group $RESOURCE_GROUP_NAME \
+    --location $RESOURCE_GROUP_LOCATION \
+    --enable-streaming-ingest true \
+    --type="None"
```

Comment:
What's --type? I don't see it in the doc page

Reply:
This is an issue reported at https://github.com/Azure/azure-cli/issues/18827. The current workaround is to set this --type= "None"' flag.

Reply:
Ah, got it.
Maybe add a comment in the script with a link to the issue?

Reply:
Link to the issue added.